data/sql/updates/pending_db_world/rev_1582810463583691504.sql

```diff
@@ -0,0 +1,18 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1582810463583691504');
+
+ALTER TABLE `battleground_template`
+CHANGE `Comment` `Comment` char(38) COLLATE 'utf8_general_ci' NOT NULL AFTER `ScriptName`;
+
+UPDATE `battleground_template` SET `Comment` = 'Alterac Valley (battleground)' WHERE `ID` = '1';
+UPDATE `battleground_template` SET `Comment` = 'Warsong Gulch (battleground)' WHERE `ID` = '2';
+UPDATE `battleground_template` SET `Comment` = 'Arathi Basin (battleground)' WHERE `ID` = '3';
+UPDATE `battleground_template` SET `Comment` = 'Nagrand Arena / Ring of Trials (arena)' WHERE `ID` = '4';
```

Comment:
These data are taken from BattlemasterList.dbc, and ID 4 is "Nagrand Arena".

Reply:
https://wow.gamepedia.com/Ring_of_Trials it's just a comment meant to inform the player, when i researched it i couldn't find "nagrand arena"

Reply:
so i feel like my change is better xD

Reply:
could be, but I think that in part we should follow the blizz standard

Reply:
it's just a comment, and in the DBC it's the column "name" not "comment"